mx-platform-aspnet-core/mx-platform-aspnet-core.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="dotenv.net" Version="3.1.1" />
+    <PackageReference Include="Json.NET" Version="1.0.33" />
+    <PackageReference Include="JsonSubTypes" Version="1.8.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Polly" Version="7.0.1" />
```

Comment:
I followed the instructions in the README to run this & I get a build error unless I change this to `Version="7.2.2"`

```
error: NU1605: Detected package downgrade: Polly from 7.2.2 to 7.0.1. Reference the package directly from the project to select a different version. 
error:  mx-platform-aspnet-core -> MX.Platform.CSharp 0.8.1 -> Polly (>= 7.2.2) 
error:  mx-platform-aspnet-core -> Polly (>= 7.0.1)
```

Reply:
I was able to reproduce it right after executing `dotnet add package MX.Platform.CSharp`. Thank you Sunita!